src/schema/schema_alter.c

```diff
@@ -91,6 +111,172 @@ __alter_object(WT_SESSION_IMPL *session, const char *uri, const char *newcfg[])
     return (__alter_apply(session, uri, newcfg, WT_CONFIG_BASE(session, object_meta)));
 }
 
+/*
+ * __alter_get_object_id_range --
+ *     Get current and oldest object IDs for the tiered object.
+ */
+static int
+__alter_get_object_id_range(WT_SESSION_IMPL *session, WT_TIERED *tiered, const char *uri,
+  uint32_t *pcurrent_id, uint32_t *poldest_id)
+{
+    WT_CONFIG_ITEM cval;
+    WT_DECL_RET;
+    uint32_t current_id, oldest_id;
+    char *value;
+
+    value = NULL;
```

Comment:
Also most functions that set pointers usually initialize them immediately. I suggest:
*current_idp = *oldest_idp = WT_TIERED_OBJECTID_NONE;